crypto/evp/evp_local.h

```diff
@@ -128,8 +128,10 @@ struct evp_keymgmt_st {
     /* Import and export routines */
     OSSL_FUNC_keymgmt_import_fn *import;
     OSSL_FUNC_keymgmt_import_types_fn *import_types;
+    OSSL_FUNC_keymgmt_import_types2_fn *import_types2;
```

Comment:
A question for other reviewers - do we want to use the _ex notation here?

Reply:
Yes. I think we do want the _ex notation.